scripts/start_fabric.sh

```diff
@@ -60,24 +65,69 @@ check_if_cc_artifacts_exists()
     return $exists
 }
 
-if [ -f "$MINIFAB" ]; then
-    echo "minifab is already installed in $MINIFAB"
+while getopts "w:udch" OPTCHAR ; do
+    case $OPTCHAR in
+        u )
+            START_FABRIC=1
+            if [[ $STOP_FABRIC == 1 || $CLEAN_UP_WORK_DIR == 1 ]]; then
+                echo "Either one of option should be spefied [u|d|c]"
```

Comment:
Change to:
`"Only one of these options should be specified: [u|d|c]"`

Reply:
Done